### PR TITLE
Document how to find the jar library versions for carrier/geocoding/prefixmapper

### PR DESCRIPTION
**Pull request by keghani**
Inspired by #1645
b/36428464

---
This change is [Reviewable](https://reviewable.io/reviews/googlei18n/libphonenumber/1669)

### CODE REVIEW

File: FAQ.md [hunk 1]
```diff
@@ -255,6 +255,24 @@ The metadata binary files can be generated using the ant build rules
...
+version of libphonenumber, follow these steps:
+
+*   Go to the versioned GitHub tag, e.g.
+    https://github.com/googlei18n/libphonenumber/find/v8.3.3
+*   Type `pom.xml`. This will surface all the `pom.xml` files as they were
+    released at the chosen tag.
+*   Find the version you care about in the corresponding `pom.xml` file. For
+    example, to find the version of the carrier jar corresponding to
+    libphonenumber 8.3.3, open `java/carrier/pom.xml` at v8.3.3's search
+    results. This is `1.56`.
```
Comment (lararennie) [inline FAQ.md]: Optional: should we say if you want the geocoder or carrier jar, that you also need the prefixmapper jar?

File: FAQ.md [hunk 2]
```diff
@@ -255,6 +255,24 @@ The metadata binary files can be generated using the ant build rules
...
+libphonenumber.
+
+For the other Maven artifacts, to find the version corresponding to a given
+version of libphonenumber, follow these steps:
+
+*   Go to the versioned GitHub tag, e.g.
+    https://github.com/googlei18n/libphonenumber/find/v8.3.3
+*   Type `pom.xml`. This will surface all the `pom.xml` files as they were
+    released at the chosen tag.
+*   Find the version you care about in the corresponding `pom.xml` file. For
```
Comment (lararennie) [inline FAQ.md]: Add "inside the top-level <version> element" perhaps.

File: FAQ.md [hunk 3]
```diff
@@ -255,6 +255,24 @@ The metadata binary files can be generated using the ant build rules
...
+version](https://github.com/googlei18n/libphonenumber/releases) of
+libphonenumber.
+
+For the other Maven artifacts, to find the version corresponding to a given
+version of libphonenumber, follow these steps:
+
+*   Go to the versioned GitHub tag, e.g.
+    https://github.com/googlei18n/libphonenumber/find/v8.3.3
+*   Type `pom.xml`. This will surface all the `pom.xml` files as they were
+    released at the chosen tag.
```
Comment (lararennie) [inline FAQ.md]: Weirdly, it also finds the other xml files because they also have a p, an o and an m in the title :) Guess it doesn't matter.

Comment (keghani) [inline FAQ.md]: Yes it's weird :)

File: FAQ.md (hunk 2 above)
Comment (keghani) [inline FAQ.md]: I don't understand what this means.

File: FAQ.md (hunk 1 above)
Comment (keghani) [inline FAQ.md]: I didn't know this! Done. (coming up)

File: FAQ.md (hunk 2 above)
Comment (keghani) [inline FAQ.md]: Find the version you care about inside the top-level element in the corresponding `pom.xml` file.

OR

Find the version you care about in the corresponding `pom.xml` file inside the top-level element.

?

I guess the second IIUC

Comment (lararennie) [inline FAQ.md]: I meant the top-level element called "version" because there are many "version" elements in the file.

Comment (keghani) [inline FAQ.md]: Makes sense, thanks! Updated :)

**Pull request merged**